### PR TITLE
Add components and images on Homepage

### PR DESCRIPTION
**Pull request by hectoruch**
Update:

- Hidden sections
- Add image background on "The map" section
- Add new section

### CODE REVIEW

File: app/styles/layout/l-homepage.scss [hunk 1]
```diff
@@ -3,7 +3,9 @@
 
 .infomap {
   align-items: center;
-  background-color: $color-8;
+  background-image: url('../../assets/images/background_the_map.png');
```
Comment (tiagojsag) [inline app/styles/layout/l-homepage.scss]: we are waiting on an image from the client here. we cannot put a picture of our own map........

Comment (hectoruch) [inline app/styles/layout/l-homepage.scss]: Fix!

**Comment by tiagojsag** [comment]
delete the image please

File: app/styles/components/c-info-web.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  .text-info-web {
+    margin-left: 110px;
+    max-width: 430px;
+
+    p {
+      //change with p application .scss
+      color: $color-9;
+      font-family: $font-family-2;
+      line-height: 1.63;
+    }
```
Comment (andresgnlez) [inline app/styles/components/c-info-web.scss]: You can get rid of this :)

File: app/styles/layout/l-homepage.scss [hunk 3]
```diff
@@ -4,7 +4,6 @@
 .infomap {
   align-items: center;
   background-color: $color-8;
-  // background-image: url('../../assets/images/background_the_map.png');
   background-repeat: no-repeat;
   background-size: cover;
   display: flex;
```
Comment (andresgnlez) [inline app/styles/layout/l-homepage.scss]: Makes any sense having background properties when we are not using a background anymore?

File: app/src/components/Home.jsx [hunk 4]
```diff
@@ -22,7 +23,7 @@ class Home extends Component {
         <CaseStudySlider />
       </section>*/}
       <InfoWeb />
-      <section className={home.infomap}>
+      <section className={[home.infomap, baseStyle.wrap].join(' ')}>
```
Comment (andresgnlez) [inline app/src/components/Home.jsx]: Use `wrap` as a  container `div`, don't mix it with another classes, please.

**Comment by tiagojsag** [comment]
Something on this PR is breaking all headers on the site.

**Comment by andresgnlez** [comment]
Is this fixed, @hectoruch?

**Pull request merged**